src/Orckestra.Composer.Website/UI.Package/Typescript/Controller/AutocompleteSearchBoxVueController.ts

```diff
@@ -123,7 +125,11 @@ module Orckestra.Composer {
 
                             const results = sectionNames.map(sectionName => {
                                 const limit = this.sectionConfigs[sectionName].limit;
-                                return ComposerClient.post(`/api/search/${sectionName}?limit=${limit}`, { Query: query })
+                                let apiPath = `/api/search/${sectionName}?limit=${limit}`;
+                                if (sectionName === 'suggestcategories' && this.sectionConfigs.suggestcategories.displayCategoryPage) {
+                                    apiPath = apiPath + `&withCategoriesUrl=${this.sectionConfigs.suggestcategories.displayCategoryPage}`;
```

Comment:
apiPath = `{apiPath}&withCategoriesUrl=${this.sectionConfigs.suggestcategories.displayCategoryPage}`;